include/cuco/detail/open_addressing/kernels.cuh

```diff
@@ -182,6 +182,44 @@ CUCO_KERNEL __launch_bounds__(BlockSize) void erase(InputIt first,
   }
 }
 
+/**
+ * @brief For each key in the range [first, first + n), applies the function object `callback_op` to
+ * the copy of all corresponding matches found in the container.
+ *
+ * @tparam CGSize Number of threads in each CG
+ * @tparam BlockSize Number of threads in each block
+ * @tparam InputIt Device accessible input iterator whose `value_type` is
+ * convertible to the `key_type` of the data structure
+ * @tparam CallbackOp Type of unary callback function object
+ * @tparam Ref Type of non-owning device ref allowing access to storage
+ *
+ * @param first Beginning of the sequence of input elements
+ * @param n Number of input elements
+ * @param callback_op Function to call on every element found in the container
```

Comment:
We need to clarify that this is acting on pairs of keys/values. Something like this, in all the docstrings?
```suggestion
 * @param callback_op Function to call on every key-value pair found in the container.
 * The function is called with a cuco::pair<Key, Value> and returns void.
```

Reply:
This impl class is the backend for all of the hashing-based containers, i.e., `static_map`, `static_set`, `static_multimap`, etc. thus we have to choose a generic docstring that works for all of them. It gives us headaches too sometimes I have to admit :D

In the actual `static_map` header we could be more precise but we often just copy&paste the docstring from the impl class to make our lives easier keeping the docs up to date.

Reply:
> In the actual static_map header we could be more precise 

that's my plan for this PR as well. On the other hand, replacing `element` with `slot` is probably sufficient.